usaspending_api/api_docs/api_documentation/advanced_award_search/transaction_sum.md

```diff
@@ -0,0 +1,41 @@
+## [Transaction Sum](#transaction-sum) DEPRECIATED
+**Route:** `/api/v2/search/transaction_sum/`
+
+**Method:** `POST`
+
+This route takes keyword search fields, and returns the fields of the searched term.
```

Comment:
This does not describe what this route does.

Reply:
getting tony to update since i dont know that endpoints purpose

Reply:
removed